oracleaq/build.sbt

```diff
@@ -0,0 +1,11 @@
+import AssemblyKeys._
+
+assemblySettings
+
+assemblyOption in assembly ~= { _.copy(includeBin = true, includeScala = false, includeDependency = false) }
+
+mergeStrategy in assembly <<= (mergeStrategy in assembly) { (old) => {
+  case PathList(ps@_*) if ps.last == "HornetQUtilBundle_$bundle.class" => MergeStrategy.first
```

Comment:
Why is there explicit handling for a `HornetQ` class only in the `oracleaq` module?